book/coming_from_bash.md

```diff
@@ -69,3 +73,18 @@ $env.Path = ($env.Path | prepend 'C:\Program Files\Git\usr\bin')
 | `pwd`                                | `$env.PWD`                                                    | Display the current directory                                     |
 | `read var`                           | `let var = input`                                             | Get input from the user                                           |
 
+## History Substitutions and Default Keybindings:
+
+| Bash                                 | Nu                                                            | Task                                                              |
+| ------------------------------------ | ------------------------------------------------------------- | ----------------------------------------------------------------- |
+| `!!`                                 | `!!`                                                          | Repeat last commandline from history                              |
+| `!$`                                 | `!$`                                                          | Repeat last commandline from history                              |
```

Comment:
This description is wrong. `Get the last spatially separated token from the last command in your history.` from my blog post.

Reply:
Now that I look at these again, they don't repeat anything. They just get the info and put it on the command line. Bash repeats. Nushell just gets but doesn't execute.